exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/otlp/OtlpConfigUtil.java

```diff
@@ -154,7 +155,7 @@ public static void configureOtlpAggregationTemporality(
     }
     AggregationTemporality temporality;
     try {
-      temporality = AggregationTemporality.valueOf(temporalityStr.toUpperCase());
+      temporality = AggregationTemporality.valueOf(temporalityStr.toUpperCase(Locale.ENGLISH));
```

Comment:
this should work too, and avoids the appearance of a locale-specific behavior

```suggestion
      temporality = AggregationTemporality.valueOf(temporalityStr.toUpperCase(Locale.ROOT));
```